docs/source/notes/get_start_xpu.rst

```diff
@@ -8,68 +8,52 @@ Hardware Prerequisite
    :widths: 50 50
    :header-rows: 1
 
-   * - Validated Hardware
-     - Supported OS
-   * - Intel® Data Center GPU Max Series
-     - Linux
-   * - Intel Client GPU
-     - Windows/Linux
+   * - Supported OS
```

Comment:
By the way, why do we need to add star here?

Reply:
it is a format in rst for tables